cmd/installer/pkg/install/install.go

```diff
@@ -338,3 +343,22 @@ func (i *Installer) Install(seq runtime.Sequence) (err error) {
 
 	return nil
 }
+
+func (i *Installer) runPreflightChecks(seq runtime.Sequence) error {
+	if seq != runtime.SequenceUpgrade {
+		// pre-flight checks only apply to upgrades
+		return nil
+	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
```

Comment:
why do we use a cancellable context here?

Reply:
good question, right now there's no real reason, my only thought might be to cancel any streaming calls in progress (if we'd have any in the future)